### PR TITLE
[Workaround] Disable MIOpenConv1x1J1 on Gfx10

### PR DESCRIPTION
**Pull request by zjing14**
- Due to the compiler issue SWDEV-271887, MIOpenConv1x1J1 gets compilation failure on Navi21
- Disabled MIOpenConv1x1J1 on Navi21 until the compiler fix merged

### CODE REVIEW

**Review by atamazov — APPROVED** [review]
LGTM!

**Review by JehandadKhan — APPROVED** [review]

**Comment by atamazov** [comment]
Converted to draft to avoid spoiling docker image cache on CI. Please change this back to "normal" PR only after ending of CI moratorium (see mail). Then first merge from develop. Push the `[Ready for review]` button only after that. Thanks.

**Comment by zjing14** [comment]
CI passed.

File: src/solver/conv_ocl_dir2Dfwd1x1.cpp [hunk 1]
```diff
@@ -36,6 +36,8 @@
 /// Known since ROCm 2.9.
 #define WORKAROUND_ISSUE_2298 1
 
+#define WORKAROUND_SWDEV_271887 1
```
Comment (atamazov) [inline src/solver/conv_ocl_dir2Dfwd1x1.cpp:39]: @zjing14 On April 7 2021 https://ontrack-internal.amd.com/browse/SWDEV-271887 was set to implemented state. It needs to find out ROCm release the fix is included in, and disable the W/A starting from that release. Then we can check that the issue is gone and close the Jira.

**Pull request merged**